### PR TITLE
Make backend container alpine (Resolves #214)

### LINKED ISSUE #214: Move backend to alpine container
**Is your feature request related to a problem? Please describe.**
Now that we decoupled monitor we don't need to build bgpstream and we can move our backend image to an alpine container

### PR DESCRIPTION
**Pull request by slowr**
### Description of PR

Use smaller and more compact alpine image for backend

What component(s) does this PR affect?

- [ ] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [x] Build System

Does the PR require changes on other components? If yes, please mark the components:

- [ ] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [ ] Build System

### Related Issue

Resolves #214

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [ ] None of the above

### Checklist:

- [x] I have read the **[contributing guide](https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: backend/entrypoint [hunk 1]
```diff
@@ -17,9 +17,7 @@ start_backend() {
             exit -1
         fi
     fi
-    sed -i 's/bind 127.0.0.1/bind 0.0.0.0/g' /etc/redis/redis.conf
-    sed -i 's/notify-keyspace-events ""/notify-keyspace-events KEA/g' /etc/redis/redis.conf
-    redis-server /etc/redis/redis.conf --daemonize yes --protected-mode no
+    redis-server /etc/redis.conf --daemonize yes --protected-mode no
```
Comment (slowr) [inline backend/entrypoint]: moved sed commands to dockerfile so they are executed once

Comment (vkotronis) [reply backend/entrypoint]: ok

File: docker-compose.yaml [hunk 2]
```diff
@@ -46,7 +46,7 @@ services:
             # uncomment after Step 2 of README
             # - ./local_configs/backend/:/etc/artemis/
             # comment after Step 2 of README
-            - ./backend/supervisor.d/:/etc/supervisor/conf.d/
+            - ./backend/supervisor.d/:/etc/supervisor.d/
```
Comment (vkotronis) [inline docker-compose.yaml]: shouldn't the same be done for the monitor?

**Comment by slowr** [comment]
> 73:C 10 Aug 14:13:39.731 # Fatal error, can't open config file '/etc/redis/redis.conf'
> See semaphore build.

didn't change the testing container yet, that's the error.

File: docker-compose.yaml (hunk 2 above)
Comment (slowr) [reply docker-compose.yaml]: monitor is ubuntu because of bgpstream

Comment (vkotronis) [reply docker-compose.yaml]: noting not to forget: check also the instructions on the wiki page about these locations (if changed, they need to be updated after merging).

**Pull request merged**